tendrl.yml

```diff
@@ -7,8 +7,9 @@
 - hosts: usm_server
   remote_user: root
   vars:
+    etcd_fqdn: "{{ ansible_fqdn }}"
     etcd_ip_address: "{{ ansible_default_ipv4.address }}"
```

Comment:
With this particular setup, we just need to make sure that `lookup('dig', etcd_fqdn) == etcd_ip_address`, as long as we use IPv4 network.

Reply:
@mbukatov do you think, that it will make sense to use  ```etcd_ip_address: "0.0.0.0"```?